### ISSUE TITLE
"npm install" produces error - No "exports" main resolved in @babel/helper-compilation-targets

### ISSUE THREAD

**Issue by tmcdos** [webpack-contrib/stylelint-webpack-plugin#210]
- Operating System: Windows 7 SP1 x64
- Node Version: 13.13.0
- NPM Version: 6.13.6
- webpack Version: 4.41.5
- stylelint-webpack-plugin Version: 1.2.3

### Expected Behavior

I have cloned this repository and tried to run `npm install` - and I expected no errors.

### Actual Behavior

The following error happened:
```
Microsoft Windows [Version 6.1.7601]
Copyright (c) 2009 Microsoft Corporation.  All rights reserved.

C:\Users\IVO>z:
Z:\>cd 3
Z:\3>cd stylelint-webpack-plugin
Z:\3\stylelint-webpack-plugin>npm install

> husky@3.1.0 install Z:\3\stylelint-webpack-plugin\node_modules\husky
> node husky install

husky > Setting up git hooks
husky > Done

> core-js@2.6.11 postinstall Z:\3\stylelint-webpack-plugin\node_modules\core-js
> node -e "try{require('./postinstall')}catch(e){}"

> husky@3.1.0 postinstall Z:\3\stylelint-webpack-plugin\node_modules\husky
> opencollective-postinstall || exit 0

> stylelint-webpack-plugin@1.2.3 prepare Z:\3\stylelint-webpack-plugin
> npm run build

> stylelint-webpack-plugin@1.2.3 prebuild Z:\3\stylelint-webpack-plugin
> npm run clean

> stylelint-webpack-plugin@1.2.3 clean Z:\3\stylelint-webpack-plugin
> del-cli dist

> stylelint-webpack-plugin@1.2.3 build Z:\3\stylelint-webpack-plugin
> cross-env NODE_ENV=production babel src -d dist --ignore "src/**/*.test.js" --
copy-files

Error [ERR_PACKAGE_PATH_NOT_EXPORTED]: No "exports" main resolved in Z:\3\stylel
int-webpack-plugin\node_modules\@babel\helper-compilation-targets\package.json
    at applyExports (internal/modules/cjs/loader.js:486:9)
    at resolveExports (internal/modules/cjs/loader.js:502:23)
    at Function.Module._findPath (internal/modules/cjs/loader.js:626:31)
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:937:27)
    at Function.Module._load (internal/modules/cjs/loader.js:833:27)
    at Module.require (internal/modules/cjs/loader.js:1010:19)
    at require (internal/modules/cjs/helpers.js:72:18)
    at Object.<anonymous> (Z:\3\stylelint-webpack-plugin\node_modules\@babel\pre
set-env\lib\debug.js:8:33)
    at Module._compile (internal/modules/cjs/loader.js:1121:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1141:10) {
  code: 'ERR_PACKAGE_PATH_NOT_EXPORTED'
}
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! stylelint-webpack-plugin@1.2.3 build: `cross-env NODE_ENV=production ba
bel src -d dist --ignore "src/**/*.test.js" --copy-files`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the stylelint-webpack-plugin@1.2.3 build script.
npm ERR! This is probably not a problem with npm. There is likely additional log
ging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     z:\temp\npm\_logs\2020-04-22T06_45_06_051Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! stylelint-webpack-plugin@1.2.3 prepare: `npm run build`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the stylelint-webpack-plugin@1.2.3 prepare script.
npm ERR! This is probably not a problem with npm. There is likely additional log
ging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     z:\temp\npm\_logs\2020-04-22T06_45_06_085Z-debug.log

Z:\3\stylelint-webpack-plugin>node -v
v13.13.0

Z:\3\stylelint-webpack-plugin>npm -v
6.13.6

Z:\3\stylelint-webpack-plugin>
```
![shot-1](https://user-images.githubusercontent.com/19253115/79950730-2a627f80-8480-11ea-8bdf-9616ac9b100b.png)

### Code

```js
// webpack.config.js
// If your code blocks are over 20 lines, please paste a link to a gist
// (https://gist.github.com).
NOT RELEVANT
```

### How Do We Reproduce?

Just clone the repository and try to run `npm install`

**Comment by evilebottnawi** [comment]
Please regenerate your lock file, problem was on babel side
